library/core/src/char/methods.rs

```diff
@@ -279,6 +279,7 @@ impl char {
     /// '1'.is_digit(37);
     /// ```
     #[stable(feature = "rust1", since = "1.0.0")]
+    #[doc(alias = "isxdigit")]
```

Comment:
This one is moderately iffy because you have to pass a radix of 16, but it's close enough.